_data/shared_hosts.yml

```diff
@@ -143,6 +143,14 @@
   php54: ??
   default: 5.4.??
 
+- name: "Kinghost"
```

Comment:
Gonna check it out. Also, things might have changed with all the (deserved) hype over PHP7's release.